Formula/g/geni.rb

```diff
@@ -0,0 +1,19 @@
+class Geni < Formula
+  desc "Database CLI migration tool"
+  homepage "https://github.com/emilpriver/geni"
+  url "https://github.com/emilpriver/geni/archive/refs/tags/v0.0.1.tar.gz"
+  version "0.0.1"
```

Comment:
This isn't needed when it's clear from the url.
```suggestion
```

Reply:
Hey, got ya. It will be removed. Thank you ☺️